sympy/ntheory/generate.py

```diff
@@ -1005,3 +1005,34 @@ def compositepi(n):
     if n < 4:
         return 0
     return n - primepi(n) - 1
+
+def nprime(nth, kth=1):
```

Comment:
The parameters should just be `n` and `k`.

Reply:
@asmeurer With parameters being just `n` and `k` people may get confused like you did https://github.com/sympy/sympy/pull/20830#issuecomment-818309003